src/app/features/user/my-products/components/product-dialog/product-dialog.component.html

```diff
@@ -41,7 +41,8 @@
 
       <mat-form-field>
         <mat-label>Product images</mat-label>
-        <ngx-mat-file-input formControlName="imageFiles" required multiple accept="image/*"></ngx-mat-file-input>
+        <ngx-mat-file-input formControlName="imageFiles" required multiple accept="image/*"
+                            [errorStateMatcher]="matcher"></ngx-mat-file-input>
```

Comment:
What is it doing?

Reply:
It is control when error messages are shown. I put it in the whole form but forgot this one